### PR TITLE
Feat/implement docs-1

### PR DESCRIPTION
**Pull request by khushi-1305**

### CODE REVIEW

File: docs/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,22 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { GetStartedComponent } from './get-started/get-started.component';
 
-const routes: Routes = [];
+const routes: Routes = [
```
Comment (izahirclemencia) [inline docs/src/app/app-routing.module.ts:5]: Why do we need routes here?

File: docs/src/app/app.module.ts [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { InputRepoComponent } from './input-repo/input-repo.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    GetStartedComponent,
```
Comment (izahirclemencia) [inline docs/src/app/app.module.ts:17]: Where are these components and are they required?

Comment (khushi-1305) [reply docs/src/app/app.module.ts:17]: Yes. These components are added in another pull request for better readability and understanding

File: docs/src/app/app-routing.module.ts (hunk 1 above)
Comment (khushi-1305) [reply docs/src/app/app-routing.module.ts:5]: We need to provide at least one route to the application. The application redirects to get-started by default.

File: docs/src/app/app.module.ts (hunk 2 above)
Comment (izahirclemencia) [reply docs/src/app/app.module.ts:17]: Do not include them in this pull request.

**Review by izahirclemencia — APPROVED** [review]

**Pull request merged**